DuckDuckGo/MainViewController.swift

```diff
@@ -346,11 +346,14 @@ class MainViewController: UIViewController {
     }
 
     func loadUrlInNewTab(_ url: URL) {
+        print(#function, "***")
```

Comment:
is this output for debug purposes?(maybe change to os_log) or did you forget to remove it?